app/src/main/java/paulscode/android/mupen64plusae/jni/CoreTypes.java

```diff
@@ -69,6 +69,10 @@ enum m64p_core_param {
         M64CORE_EMU_STATE,
         M64CORE_VIDEO_MODE,
         M64CORE_SAVESTATE_SLOT,
+        M64CORE_AUDIO_INIT,
```

Comment:
New enums should probably be added at the end of the list to stay somewhat compatible with the upstream core.